versions.json

```diff
@@ -23,11 +23,6 @@
       "branch": "8.16",
       "previousMajor": true
     },
-    {
```

Comment:
We're supporting 8.15 until 8.17 is released.

Reply:
yeah, you're right. Just fixed that